module.yml

```diff
@@ -6,3 +6,10 @@ type: 'node'
 script: 'index.js'
 require:
   mongodb
+config:
```

Comment:
How come this is defined here? It should be defined by each client

Reply:
That's for the mhub message.